packages/playwright-core/src/server/dispatchers/localUtilsDispatcher.ts

```diff
@@ -146,7 +146,6 @@ async function urlToWSEndpoint(progress: Progress, endpointURL: string): Promise
     return new Error(`Unexpected status ${response.statusCode} when connecting to ${fetchUrl.toString()}.\n` +
         `This does not look like a Playwright server, try connecting via ws://.`);
   });
-  progress.throwIfAborted();
```

Comment:
Why do we pass timeout to fetchData above instead of passing progress?

Reply:
Yeah, there is a TODO in my list to pass progress there. I'll follow up.